### PR TITLE
Update doxygen comments in OTA agent header

### PR DESCRIPTION
**Pull request by tgsong**
Description
-----------

Update doxygen comments in OTA agent header.

Checklist:
----------

- [x] I have tested my changes. No regression in existing tests.
- [x] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h [hunk 1]
```diff
@@ -668,23 +672,24 @@ OTA_Err_t OTA_SetImageState( OTA_ImageState_t eState );
...
  * @return kOTA_Err_None if successful, otherwise an error code prefixed with 'kOTA_Err_' from the
  * list above.
  */
 OTA_Err_t OTA_CheckForUpdate( void );
 
-/* @brief Suspend OTA agent oeprations .
- *
- * @param[in]
+/**
+ * @brief Suspend OTA agent oeprations .
```
Comment (gkwicker) [inline libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h:684]: spelling: _operations_

File: libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h [hunk 2]
```diff
@@ -668,23 +672,24 @@ OTA_Err_t OTA_SetImageState( OTA_ImageState_t eState );
...
  *
  * @return kOTA_Err_None if successful, otherwise an error code prefixed with 'kOTA_Err_' from the
  * list above.
  */
 OTA_Err_t OTA_Suspend( void );
 
-/* @brief Resume OTA agent oeprations .
+/**
+ * @brief Resume OTA agent oeprations .
```
Comment (gkwicker) [inline libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h:692]: spelling: _operations_

File: libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h (hunk 1 above)
Comment (tgsong) [reply libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h]: Fixed.

File: libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h (hunk 2 above)
Comment (tgsong) [reply libraries/freertos_plus/aws/ota/include/aws_iot_ota_agent.h]: Fixed.

**Pull request merged**